roles/nginxplus/tasks/modules/install-xslt.yml

```diff
@@ -1,18 +1,18 @@
 ---
-- name: "Install: Install NGINX Open Source XSLT Module"
-  apt:
+- name: nginxplus | Install NGINX Open Source XSLT Module
+  ansible.builtin.apt:
     name: nginx-module-xslt
     state: present
   when: nginx_type == "opensource"
 
-- name: "Install: Install NGINX Plus XSLT Module"
-  apt:
+- name: nginxplus | Install NGINX Plus XSLT Module
+  ansible.builtin.apt:
     name: nginx-plus-module-xslt
     state: present
   when: nginx_type == "plus"
 
-- name: "Setup: Load NGINX XSLT Module"
-  lineinfile:
+- name: nginxplus | Load NGINX XSLT Module
+  ansible.builtin.lineinfile:
```

Comment:
We've got a lot of `lineinfile` tasks for the Nginx config file. How about using a template instead of editing and re-editing the file on the server?